aggregator/fetchers/factory.go

```diff
@@ -13,18 +13,18 @@ type MaiarTokensPair struct {
 }
 
 // NewPriceFetcher returns a new price fetcher of the type provided
-func NewPriceFetcher(fetcherName string, responseGetter aggregator.ResponseGetter, maiarTokensMap map[string]MaiarTokensPair) (aggregator.PriceFetcher, error) {
+func NewPriceFetcher(fetcherName string, responseGetter aggregator.ResponseGetter, graphqlGetter aggregator.GraphqlGetter, maiarTokensMap map[string]MaiarTokensPair) (aggregator.PriceFetcher, error) {
```

Comment:
do we need a "disabled" implementation for the aggregator.GraphqlGetter?

Reply:
I don`t think we really need it atm. Indeed it would be better to have it in the future (also for HttpResponseGetter)

Reply:
ok